civic-auth/integration/react.md

```diff
@@ -92,6 +92,68 @@ This component is context-dependent. If the user is logged in, it will show thei
 
 <figure><img src="../.gitbook/assets/image (1).png" alt=""><figcaption><p>The user button</p></figcaption></figure>
 
+#### Customizing the User Button
+You can customize the styling of the user button by adding either a `className` or `style` property to the UserButton component when declaring it e.g.
+Using a className:
+{% code title="CustomUserButtonClassName.ts" %}
+```css
+.my-button-container .login-button {
+  color: red;
+  background-color: blue;
+  border: 3px solid #6b7280;
+}
+```
+```typescript
+import { UserButton, CivicAuthProvider } from "@civic/auth-web3/react";
+
+export function TitleBar() {
+  return (
+    <div className="flex justify-between items-center">
+      <h1>My App</h1>
+      <div className="my-button-container">
+        <UserButton className="login-button" />
+      </div>
+    </div>
+  );
+};
+```
+Note the use of a _specific_ class name declaration in the .css file. This is necessary to ensure that the styles in the imported css className take precedence over internal styles without the use of the discouraged `!important` directive.
```

Comment:
maybe add an example here of what wouldn't work?

Reply:
done